internal/pkg/describe/status.go

```diff
@@ -80,14 +89,15 @@ type AppRunnerStatusDescriber struct {
 
 // ecsServiceStatus contains the status for an ECS service.
 type ecsServiceStatus struct {
-	Service      awsECS.ServiceStatus
-	Tasks        []awsECS.TaskStatus      `json:"tasks"`
-	Alarms       []cloudwatch.AlarmStatus `json:"alarms"`
-	StoppedTasks []awsECS.TaskStatus      `json:"stoppedTasks"`
+	Service           awsecs.ServiceStatus
+	Tasks             []awsecs.TaskStatus      `json:"tasks"`
+	Alarms            []cloudwatch.AlarmStatus `json:"alarms"`
+	StoppedTasks      []awsecs.TaskStatus      `json:"stoppedTasks"`
+	TasksTargetHealth []taskTargetHealth       `json:"targetsHealth"`
```

Comment:
What do you think of:
```suggestion
	TargetGroups struct{
	     Health []taskTargetHealth  `json:"health"`
	} `json:"targetGroups"`
```